configure-travis-env.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+red=$'\e[1;31m'
```

Comment:
I'll remove the unused ones of these when removing the changes introduced for testing.